samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,54 @@
+Testusb
+#######
+
+native_posix configuration
+**************************
+
+Build testusb with native_posix configuration:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/testusb
+   :board: native_posix
+   :goals: build
+   :compact:
+
+Run testusb application with:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/usb/testusb
+   :board: native_posix
+   :goals: run
+   :compact:
+
+In a terminal window following comands to list USB device:
+
+.. code-block:: console
+
+   $ usbip list -r 127.0.0.1
+   Exportable USB devices
+   ======================
+    - 127.0.0.1
+           1-1: unknown vendor : unknown product (2fe3:0100)
+              : /sys/devices/pci0000:00/0000:00:01.2/usb1/1-1
+              : (Defined at Interface level) (00/00/00)
+              :  0 - Vendor Specific Class / unknown subclass / unknown protocol (ff/00/00)
+
+In a terminal window following comands to attach USB device:
```

Comment:
Same change here (and a spelling fix):

    In a terminal window, run the following commands to attach USB devices: